src/nodeTypes.ml

```diff
@@ -0,0 +1,28 @@
+module ArrayBuffer = struct
+  type t
+end
```

Comment:
It was an inertia after resolving cyclic dependency and result of not clear understanding. I'm going to eliminate NodeTypes at all and place types in their relevant modules.